### ISSUE TITLE
./build_all --clean_debug not working

### ISSUE THREAD

**Issue by zhucaoxiang** [PrincetonUniversity/STELLOPT#71]
I am trying to use `` and got the following output.
```
Beginning build of STELLOPT with options: --clean_debug
make: Nothing to be done for `clean_debug'.
```

Basically, the optional commandline argument `--clean_debug` is treated as one of the codes, but match none. We should fix this with advanced options. Also, there might be ways to specify the `make -jN` option from the top level as well.
```shell
  if [ $# -ne 0 ]; then
    echo $@|grep -q $CODE
    if [ $? -eq 0 ]; then
      match=1  
    fi
  else
    echo $CODE
    match=1
  fi
```

**Comment by lazersos** [comment]
@zhucaoxiang I can look into it, but I only contributed to this script.  I believe @amicitas and @jonathanschilling are the original authors.

**Comment by zhucaoxiang** [comment]
Fixed in #72
